### DISCUSSION TITLE
Add some element properties to atom species

### DISCUSSION THREAD

**Discussion by yesheng22** [mir-group/allegro#40] [Q&A]
Hi，
I wish to introduce some element-dependent properties besides the atomic number when generating the initial scalar features. Can such a change be achieved by changing the config file? If possible, how to set related parameters.
Thanks

**Comment by Linux-cpp-lisp** [comment]
Hi @yesheng22 ,

You can do this by replacing `OneHotAtomEncoding` (https://github.com/mir-group/allegro/blob/main/allegro/model/_allegro.py#L59) with a custom module that outputs the one hot along with your custom properties.  This can be done with a custom extension package.  See https://github.com/mir-group/nequip-example-extension.
